test.py

```diff
@@ -373,10 +373,11 @@ class BoostTestSuite(UnitTestSuite):
 
     def _generate_cache(self) -> None:
         # Apply combined test only for test/boost
-        exe_path = pathlib.Path(self.mode, "test", self.name, 'combined_tests')
-        if self.name != 'boost' or not exe_path.exists():
+        if self.name != 'boost':
             return
         exe = path_to(self.mode, "test", self.name, 'combined_tests')
+        if not exe_path.exists():
```

Comment:
I'm confused. What is the value of exe_path now?
The assignment above to exe_path was deleted, and the line above assigns to `exe` rather than `exe_path`.

Reply:
sorry, i forgot to change `exe_path` to `exe`. should be fixed in the latest revision.